Resolve issues in API documentation (#31)
firecampapp/firecamp-doc: source/apis/impressions/query-parameters.mdx

For example, breaking down the URL (Uniform Resource Locator) https://jsonplaceholder.typicode.com/posts?id=12&name=vipulgupta2048

The protocol is *https*  
The domain name is *jsonplaceholder.typicode.com*  
The path to the resource (Path Parameter) is */posts*  

> Finally, the query parameters being *id=12&name=vipulgupta2048*  

Reviewer: I think bold would look better. Go bold!

Query Parameters are critical in sorting or filtering through resources for developers to find the right data. One can specify query parameters easily using the `Params` tab in the request panel of the API app.

> PLACEHOLDeR FOR IMAGE of adding params

> Adding query parameters at any place in the API app will lead to them being added to the URL bar instantly.

Reviewer: ```suggestion
Adding query parameters in the API app will result in them being added instantly to the URL present in the URL bar as parameters
```
